bazel-java-deps.bzl

```diff
@@ -155,7 +155,7 @@ def install_java_deps():
             "org.apache.commons:commons-text:1.4",
             "org.awaitility:awaitility:3.1.6",
             "org.checkerframework:checker:2.5.4",
-            "org.flywaydb:flyway-core:7.13.0",
```

Comment:
do we need to upgrade Flyway as well?

Reply:
yes, the old flyway version is incompatible with the latest h2.